client/tests/webdriver/customFieldsSpecs/showTask.spec.js

```diff
@@ -30,4 +30,44 @@ describe("Show task page", () => {
       expect(await question1AssessmentWeekly.isExisting()).to.be.true
     })
   })
+
+  describe("When on the show page of a task", () => {
```

Comment:
Change this test to "When in the show page of a task with a parent task",  load a task with a parent tasks, and do the appropriate assertion.